health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/CampaignHome.js

```
@@ -48,21 +139,21 @@ const CampaignHome = () => {
           }}
         >
           <NewWindow />
-          <div className={"descStyle"}>{t("HCM_CREATE_NEW_CAMPAIGN")}</div>
+          <div className={"descStyle "}>{t("HCM_CREATE_NEW_CAMPAIGN")}</div>
         </div>
         <div
           className={"cardStyle disabledCard"}
           onClick={() => {
             // Add functionality for importing existing campaign
           }}
         >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)